inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/manager/EventsManager.kt

```diff
@@ -30,7 +31,7 @@ internal object EventsManager {
     ) {
         val isUserUpdated = accountRepo.updateUserInfo()
         // Caching events locally.
-        event.setUserUpdated(isUserUpdated || isUpdated)
+        event.setShouldNotClear(isUserUpdated || isUpdated || PingResponseMessageRepository.isInitialLaunch)
```

Comment:
Previously `isUserUpdated` made that event be deleted or preserved?

Reply:
Preserved (not cleared)